build/build.proj

```diff
@@ -242,12 +242,14 @@
     BuildVS15 runs the NoVSIX targets before this one.
     ============================================================
   -->
-  <Target Name="BuildVS15VSIX" AfterTargets= "BuildVS15" Condition=" '$(IsXPlat)' != 'true' ">
+  <Target Name="BuildVS15VSIX" AfterTargets= "BuildVS15" Condition=" '$(IsXPlat)' != 'true' AND '$(BuildRTM)' != 'true'">
     <Message Text="Building the VSIX for VS15" Importance="high" />
 
     <MSBuild Projects="$(VSIXProject)"
              Targets="Build"
              Properties="$(CommonMSBuildProperties);
+                         BuildProjectReferences=false;
+                         IsVsixBuild=true;
```

Comment:
no, i am keeping the VS14 as is, and VSTS isn't going to be building VS14 most likely.